### DISCUSSION TITLE
ES-lint Failures

### DISCUSSION THREAD

**Discussion by alunavat** [CSC510-Group-5/URL-Shortner#63] [General]
Saw some ES lint failures, are these actual ones? Are we seeing the report somewhere? @deep-mm

**Comment by deep-mm** [comment]
@alunavat Yes, I will correct the ESLint failures as suggested
Another suggestion, we can actually publish the ESLint results to Security CodeScans tab if we want to track the ones which are open and ones which get closed

**Reply by alunavat** [reply]
Yeah, that is a good idea.

**Comment by deep-mm** [comment]
@alunavat fixed these issues in the latest PR #65 

Take a look once

**Reply by alunavat** [reply]
Looks good. Thank you @deep-mm
